web/src/components/navbar/NavBar.tsx

```diff
@@ -13,6 +14,7 @@ type Props = {
   showSidebar?: boolean;
   hideMiniRoadmap?: boolean;
   logoOnly?: "NAVIGATOR_LOGO" | "NAVIGATOR_MYNJ_LOGO" | undefined;
+  previousBusiness?: Business | undefined;
```

Comment:
dust - i get why `useMockUserData({});` was added to `NavBar.test.tsx` - that being said, it feels weird passing in the the `business` object while also using the userData hook to grab the `business` within individual components.

I think it might be better to have a helper function that takes a business as an argument and returns the `id` and this prop takes that id. 
`<Navbar prevBusId={getBusinessId(business)} />`

Reply:
I think this is a valid point. I was just following the pattern established in `ReturnToPreviousBusinessBar.tsx` but we only require an id in this case so I can make that change.